src/sentry/integrations/vercel/integration.py

```diff
@@ -256,7 +256,7 @@ def create_env_var(self, client, vercel_project_id, key, value, type):
         data = {
             "key": key,
             "value": value,
-            "target": ["production"],
+            "target": ["production", "preview", "development"],
```

Comment:
Would having `development` work well here? Vercel itself wouldn't be running anything on `development`, so this would be environment variables that are pulled down for local use. 

In localhost, you might run code in files that aren't committed yet. What would happen in terms of releases then? 

In my perspective, an error without a source map for JavaScript should not exist. Would this change not make more of those?

Reply:
Hm, lemme think about this. When running `vercel dev`, we are building/running Next.js in development mode where we don't upload source maps anyhow. The errors thrown in dev mode should be dynamically source mapped - we built this in https://github.com/getsentry/sentry-javascript/pull/7294. So I think you have a point here.

BUT, people are not exclusively deploying Next.js apps to Vercel. I think it might be beneficial to have the env vars available if you have added the integration for different kinds of setups. For example, we're currently working on a sveltekit SDK which might make use of the variables.

What do you think?

Reply:
There are ways in which it makes a lot of sense. Having the DSN in .env seems like a good thing for example. It makes sense to have more of the data locally for other types of projects I too I guess. My only concern is that one of your env vars is `VERCEL_GIT_COMMIT_SHA` That wouldn't work locally. Would it be somehow pulled as well? I think it might have to be excluded? 

But as long as the default setup from `npx @sentry/wizard -s -i nextjs` can handle that, and with also when #5734 is fixed, I think it's fine to have them on `development` as well.


I personally don't use `vercel dev`, so I don't know what that does.

Reply:
> My only concern is that one of your env vars is VERCEL_GIT_COMMIT_SHA That wouldn't work locally. Would it be somehow pulled as well? I think it might have to be excluded?

Ah, you're right. Gonna exclude this from the development target. 👍

Reply:
Actually not even `SENTRY_AUTH_TOKEN` is needed locally, and it is probably even dangerous to put it onto users' machines.

Reply:
Actually actually no env vars besides the DSN are useful in development

Reply:
Not SENTRY_PROJECT or SENTRY_ORG even?

Reply:
Not really, they're only used for uploading source maps, which we don't do in dev mode.

Reply:
Sounds good then! Thanks for discussing.